cmd/cli.go

```diff
@@ -160,14 +160,14 @@ func (cli *CLI) Run(args []string) int {
 
 	issues := tflint.Issues{}
 
-	provider := rules.NewProvider()
+	ruleset := rules.NewRuleSet()
 	for _, runner := range runners {
-		providerIssues, err := provider.Check(runner)
+		ret, err := ruleset.Check(runner)
```

Comment:
I would keep `issues` in the name of returned variable as it makes code more readable.